repose-aggregator/services/service-client/api/src/main/java/org/openrepose/services/serviceclient/akka/AkkaServiceClientException.java

```diff
@@ -0,0 +1,19 @@
+package org.openrepose.services.serviceclient.akka;
+
+public class AkkaServiceClientException extends Exception {
```

Comment:
Yeah, we'll add the others as we need them, because they are probably not getting coverage right now either.